src/Facebook/FacebookRequest.php

```diff
@@ -249,7 +249,7 @@ public function execute()
 
     static::$requestCount++;
 
-    $etagHit = 304 == $connection->getResponseHttpStatusCode();
+    $etagHit = (304 === $connection->getResponseHttpStatusCode());
```

Comment:
The parentheses around this statement is redundant and harder to read so it should be like it was before. The strict `===` checking is OK though.